setup.py

```diff
@@ -31,10 +30,10 @@ def read(f):
         "Intended Audience :: Developers",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
```

Comment:
When the support of 3.5 was officially ended? 🤔

Reply:
[Python End Of Life](https://endoflife.date/python)

3.6 - Ended 1 month and 2 weeks ago (23 Dec 2021)
3.5 - Ended 1 year and 4 months ago (13 Sep 2020)